hbase-common/src/main/java/org/apache/hadoop/hbase/io/ByteBuffAllocator.java

```diff
@@ -129,6 +145,13 @@
    * @return ByteBuffAllocator to manage the byte buffers.
    */
   public static ByteBuffAllocator create(Configuration conf, boolean reservoirEnabled) {
+    if (conf.get(DEPRECATED_BUFFER_SIZE_KEY) != null
+        || conf.get(DEPRECATED_MAX_BUFFER_COUNT_KEY) != null) {
+      LOG.warn("The config keys {} and {} are deprecated now, instead please use {} and {}. In "
+            + "future release we will remove the two deprecated configs.",
+        DEPRECATED_BUFFER_SIZE_KEY, DEPRECATED_MAX_BUFFER_COUNT_KEY, BUFFER_SIZE_KEY,
+        MAX_BUFFER_COUNT_KEY);
+    }
     int poolBufSize = conf.getInt(BUFFER_SIZE_KEY, DEFAULT_BUFFER_SIZE);
```

Comment:
Shouldn't we use ``conf.getInt(DEPRECATED_BUFFER_SIZE_KEY)``, and put in if branch above?

The old configs user sets still take no effect.

Reply:
The same to ``DEPRECATED_MAX_BUFFER_COUNT_KEY``

Reply:
The same to ``DEPRECATED_MAX_BUFFER_COUNT_KEY``

Reply:
> Shouldn't we use conf.getInt(DEPRECATED_BUFFER_SIZE_KEY), and put in if branch above?

No need to do that,  because the Configuration#getInt have implement this logic internal, you can see the code.

Reply:
Checked.